icalendar.gemspec

```diff
@@ -28,20 +28,19 @@ ActiveSupport is required for TimeWithZone support, but not required for general
 
   s.add_dependency 'ice_cube', '~> 0.16'
 
-  s.add_development_dependency 'rake', '~> 12.0'
+  s.add_development_dependency 'rake', '~> 13.0'
   s.add_development_dependency 'bundler', '~> 2.0'
 
   # test with all groups of tzinfo dependencies
   # tzinfo 2.x
   # s.add_development_dependency 'tzinfo', '~> 2.0'
-  # s.add_development_dependency 'tzinfo-data', '~> 1.2018'
+  # s.add_development_dependency 'tzinfo-data', '~> 1.2020'
   # tzinfo 1.x
-  s.add_development_dependency 'activesupport', '~> 5.2'
-  s.add_development_dependency 'i18n', '~> 1.1'
+  s.add_development_dependency 'activesupport', '~> 6.0'
```

Comment:
Should be possible to use icalendar with rails prior to v6

Reply:
It is, only gems defined with `s.add_dependency` are required for a user to install it. This just defines what is loaded for tests.

Ideally we'd have a test harness that is set up to test with the various combos of tzinfo (not installed, 0.x, 1.x, 2.x) and activesupport (not installed, 5.x, 6.x, maybe also 4.x?) that we allow, but that's outside of the scope of this PR

Reply:
Indeed. Sorry my mistake.